src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -433,12 +433,19 @@ void
 OverlayImpl::setupValidatorKeyManifests (BasicConfig const& config,
                                          DatabaseCon& db)
 {
-    auto const loaded = manifestCache_.loadValidatorKeys (
+    auto loaded = manifestCache_.loadValidatorKeys (
+        config.section ("validators"),
+        journal_);
+
+    if (!loaded)
+        Throw<std::runtime_error> ("Unable to load keys from [validators]");
+
```

Comment:
I chose to convert to a string right away because we'd end up doing a string append in the exception message, and the parameter to `config.section` is also a string. I'm not sure what the status is with `"string literal"s`. If we could use that then I'd go back to auto.